oracle/cloud-config.yml

```diff
@@ -0,0 +1,64 @@
+azs:
+- name: z1
+  cpi: oracle-cpi-ref
+  cloud_properties:
+    availability_domain: ((ad1))
+- name: z2
+  cpi: oracle-cpi-ref
+  cloud_properties:
+    availability_domain: ((ad2))
```

Comment:
In our other CPIs we list several available zones, but internally refer them to the same IaaS availability zone. This simplifies the "getting started" process for users, without requiring them to setup or know multiple azs and networks.

Would you be okay with changing this to follow that convention for `azs` and `subnets` here as well?

Reply:
Sure